### PR TITLE
Brushed output

### PR DESCRIPTION
**Pull request by sahil-kale**

### CODE REVIEW

**Comment by sahil-kale** [comment]
Ensure that PWM input is approved first!

File: Core/Inc/PWMinput.hpp [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#ifndef PWM_INPUT_HPP
```
Comment (AadityaChaudhary) [inline Core/Inc/PWMinput.hpp:1]: why have you defined the input stuff in this output or?

File: Core/Src/brushed_output.cpp [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+{
+	if(!s_instance)
+	{
+		s_instance = new BrushedMotorDriver();
+	}
+
+	return s_instance;
+}
+
+bool BrushedMotorDriver::set(double speed)
```
Comment (AadityaChaudhary) [inline Core/Src/brushed_output.cpp:39]: logic looks dubious, could you leave a comment explaining this function and how it handles forward/reverse speeds?

File: Core/Src/PWMinput.cpp [hunk 3]
```diff
@@ -0,0 +1,71 @@
+#include "tim.h"
```
Comment (AadityaChaudhary) [inline Core/Src/PWMinput.cpp:1]: why was this needed in this pr? to resolve merge issues with git?

**Review by AadityaChaudhary — CHANGES_REQUESTED** [review]

File: Core/Src/PWMinput.cpp (hunk 3 above)
Comment (sahil-kale) [reply Core/Src/PWMinput.cpp:1]: Tim is needed for the timer object definitons

**Pull request closed without merging**